leftwm-core/src/handlers/command_handler.rs

```diff
@@ -598,17 +598,12 @@ fn focus_window_top(state: &mut State, toggle: bool) -> Option<bool> {
 fn focus_workspace_change(state: &mut State, val: i32) -> Option<bool> {
     let current = state.focus_manager.workspace(&state.workspaces)?;
     let workspace = helpers::relative_find(&state.workspaces, |w| w == current, val, true)?.clone();
-    state.focus_workspace(&workspace);
+    let result = state.focus_workspace(&workspace);
```

Comment:
I am wondering, if we really need to have this before the following `if`. Like this it feels a bit weird to define the `result` before checking for diverging behaviour.
But if we need to keep this order, maybe a little commentary or a different name might help in the future.

Reply:
Actually, do we even need to call `focus_workspace` if the focus behavior is sloppy? shouldn't moving the cursor trigger the the call

Reply:
I guess this was once introduced as a bug-fix I guess, but this means we need the documentation, why it's there even more.
@AethanFoot may we have some insight from you, as I guess you might have touched this line with #356

Reply:
Ah, I think I get it now: this moves the cursor to the center of the focused window in `Sloppy` in order to not trigger a refocus if mouse is moved after the workspace focus changed was called by command and not by mouse movement.

Reply:
Yes, but I think it should move it to the center of the last focused window instead, otherwise it can be moved to a position that does not have any window, e.g. if you have 2 windows in `EvenHorizontal` or `EvenVertical`, the cursor get positioned in the margin between 2 windows, cf my [comment](https://github.com/leftwm/leftwm/pull/642#discussion_r777214932) on the other thread

Reply:
I think that action is there in case there is no window on the workspace. So as the window focusing has moved the focus_workspace can be after, you could also check if there is a window on the workspace to prevent the extra action.

Reply:
We should probably add the handle_focus part within the focus_window function which would solve these issues and reduce the code in command_handler.